Assets/MixedRealityToolkit/_Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -13,59 +15,64 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
     /// <remarks>One definition should exist for each physical device input, such as buttons, triggers, joysticks, dpads, and more.</remarks>
     /// </summary>
     [Serializable]
-    public struct InteractionMapping
+    public struct MixedRealityInteractionMapping : IMixedRealityInteractionMapping, ISerializationCallbackReceiver
```

Comment:
Please review with the changes in the other PR, shouldn't be needed as it should not be using an interface